src/ogr/geometry.jl

```diff
@@ -1,3 +1,32 @@
+
+# This is friendly type-pyracy on `convert` for GeoFormatTypes types
+Base.convert(::Type{<:Geometry}, source::GFT.AbstractWellKnownText) = 
+    fromWKT(GFT.val(source))
+Base.convert(::Type{<:Geometry}, source::GFT.WellKnownBinary) = 
+    fromWKB(GFT.val(source))
+Base.convert(::Type{<:Geometry}, source::GFT.GeoJSON) = 
+    fromJSON(GFT.val(source))
+Base.convert(::Type{<:Geometry}, source::GFT.GML) = 
+    fromGML(GFT.val(source))
+
+Base.convert(::Type{<:GFT.AbstractWellKnownText}, source::AbstractGeometry) = 
+    GFT.WellKnownText(GFT.Geom(), toWKT(source))
+Base.convert(::Type{<:GFT.WellKnownBinary}, source::AbstractGeometry) = 
+    GFT.WellKnownBinary(GFT.Geom(), toWKB(source))
+Base.convert(::Type{<:GFT.GeoJSON}, source::AbstractGeometry) = 
+    GFT.GeoJSON(toJSON(source))
+Base.convert(::Type{<:GFT.GML}, source::AbstractGeometry) = 
+    GFT.GML(GFT.Geom(), toGML(source))
+Base.convert(::Type{<:GFT.KML}, source::AbstractGeometry) = 
+    GFT.KML(toKML(source))
+
+# Convert to Geometry, then to the target format. There may be
+# a more optimal method, but this is a simple, high-level first pass.
+# The Geom trait is needed to separate out convert for CRS for WellKnownText
+# and GML, which may contain both. It is handled in GeoFormatTypes.
+Base.convert(format::Type{<:GFT.GeoFormat}, mode::GFT.Geom, source::GFT.GeoFormat) =
+    convert(format, convert(Geometry, source))
```

Comment:
Can we move these methods into a `conversions.jl` file under [`src/base`](https://github.com/yeesian/ArchGDAL.jl/tree/master/src/base)?

Reply:
You should make the comment above the top-level comment for that file, and explain it to developers in a way that doesn't require an understanding of GeoFormatTypes.

Reply:
conversions.jl is a good idea.

Reply:
this is all in convert.jl, not sure why I didn't call it conversions.jl lol